openghg/data/config/objectstore/defaults.json

```diff
@@ -0,0 +1,222 @@
+            "inlet": {
+                "type": [
+                    "height"
+                ]
+            },
+            "domain": {
+                "type": [
+                    "str"
+                ]
+            },
+            "time_resolved": {
+                "type": [
+                    "bool"
+                ]
+            },
+            "high_spatial_resolution": {
+                "type": [
+                    "bool"
+                ]
+            },
+            "short_lifetime": {
+                "type": [
+                    "bool"
+                ]
+            },
+            "species": {
+                "type": [
+                    "str",
+                    "species"
+                ]
+            },
+            "met_model": {
+                "type": [
+                    "str"
+                ]
+            }
+        }
+    },
+    "flux": {
+        "required": {
+            "species": {
+                "type": [
+                    "str"
+                ]
+            },
+            "source": {
+                "type": [
+                    "str"
+                ]
+            },
+            "domain": {
+                "type": [
+                    "str"
+                ]
+            }
+        },
+        "optional": {
+            "database": {
+                "type": [
+                    "str"
+                ]
+            },
+            "database_version": {
+                "type": [
+                    "str"
+                ]
+            },
+            "model": {
+                "type": [
+                    "str"
+                ]
+            }
+        }
+    },
+    "surface": {
+        "required": {
+            "species": {
+                "type": [
+                    "str"
+                ]
+            },
+            "site": {
+                "type": [
+                    "str"
+                ]
+            },
+            "sampling_period": {
+                "type": [
+                    "str"
+                ]
+            },
+            "station_long_name": {
+                "type": [
+                    "str"
+                ]
+            },
+            "inlet": {
+                "type": [
+                    "str"
+                ]
+            },
+            "instrument": {
+                "type": [
+                    "str"
+                ]
+            },
+            "network": {
+                "type": [
+                    "str"
+                ]
+            },
+            "source_format": {
+                "type": [
+                    "str"
+                ]
+            },
+            "data_source": {
+                "type": [
+                    "str"
+                ]
+            },
+            "icos_data_level": {
+                "type": [
+                    "str"
+                ]
+            },
+            "data_type": {
+                "type": [
+                    "str"
+                ]
```

Comment:
We should be adding the new discussed keywords for "surface" here as well for surface (required with a default) - from #551 and #654 .